libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/drillDown.spec.ts

```diff
@@ -94,7 +117,7 @@ describe("Drilling", { tags: ["post-merge_integrated_bear"] }, () => {
         });
     });
 
-    describe("Advanced drill down", () => {
+    describe("Advanced drill down", { tags: ["post-merge_integrated_bear"] }, () => {
```

Comment:
It should be the basic tests. And why do you add it to  `post-merge_integrated_bear`?